### DISCUSSION TITLE
Ignore tokens and enrich the lemma graph

### DISCUSSION THREAD

**Discussion by Albertobegue** [DerwenAI/pytextrank#254] [Ideas]
Hi everyone!

It is mentioned in the project's description that enriching the lemma graph would improve TextRank's performance. I saw that showing examples of this was in the todo list of the project but I was wondering if it worked by simply adding entities to the doc before summarising? Or is it more complicated? I am particularly interested in adding hyponymy. 

And what about ignoring tokens? Some tokens are ignored depending on their POS tag in your implementation. Is it possible to ignore tokens specific to our application by tagging them? With what?

Thanks in advance for your answers!!

And thank you for this project, it is great!

**Comment by ceteri** [comment]
Thank you @Albertobegue  we're working toward these capabilities with the `kglab` integration.
